gwtquery-core/src/main/java/com/google/gwt/query/client/plugins/effects/Transform.java

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2014, The gwtquery team.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.gwt.query.client.plugins.effects;
+
+import com.google.gwt.dom.client.Element;
+import com.google.gwt.query.client.GQuery;
+import com.google.gwt.regexp.shared.MatchResult;
+import com.google.gwt.regexp.shared.RegExp;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map.Entry;
+
+/**
+ * A dictionary class with all the properties of an element transform
+ * which is able to return the correct syntax for setting the css transform
+ * property.
+ */
+public class Transform  {
+
+  private static final String TRANSFORM = "_t_";
+
+  protected static final RegExp transformRegex = RegExp.compile("^(scale([XYZ]|)|translate([XYZ]|3d)|rotate([XYZ]|3d)?|perspective|skew[XYZ]|x|y)$");
+  private static final RegExp transform3dRegex = RegExp.compile("^(rotate([XYZ]|3d)|perspective)$");
+
+  private HashMap<String, List<String>> map = new HashMap<String, List<String>>();
+
+  /**
+   * Return the Transform dictionary object of a element.
+   */
+  public static Transform getInstance(Element e) {
+    return getInstance(e, null);
+  }
+
+  public static boolean isTransform(String propName) {
```

Comment:
All code here was just moved from Transition class, so this lack of doc was previously.
I have reviewed documentation, and also moved more stuff here because makes more sense that all transform stuff is in this class.